app/policies/comment_policy.rb

```diff
@@ -0,0 +1,24 @@
+class CommentPolicy < ApplicationPolicy
+  attr_reader :user, :comment
+
+  def initialize(user, comment)
+    @user = user
+    @comment = comment
+  end
+
+  def create?
+    true
+  end
+
+  def update?
+    if user.id == @comment.author_id
+      true
```

Comment:
this can be short handed to just be ```user.id == @comment.author_id``` which will check if its true or false and return that value.

Reply:
Agreed, you can also drop the `@`. In fact, you can also drop the `id`: `user == comment.author`